app/src/main/java/org/greenstand/android/TreeTracker/database/entity/LocationEntity.kt

```diff
@@ -16,7 +16,7 @@ import androidx.room.PrimaryKey
         )
     ]
 )
-data class LocationEntity(
+data class  LocationEntity(
```

Comment:
There are a ton of files with small changes. Please avoid making changes to files that are unrelated to the goal of the PR. It makes it more likely that a merge conflict will happen.